src/rp2_common/hardware_pio/include/hardware/pio.h

```diff
@@ -647,7 +647,7 @@ static inline void sm_config_set_mov_status(pio_sm_config *c, enum pio_mov_statu
  * \return the default state machine configuration which can then be modified.
  */
 static inline pio_sm_config pio_get_default_sm_config(void) {
-    pio_sm_config c = {0};
+    pio_sm_config c = {0, 0, 0, 0};
```

Comment:
Might need an extra zero for when PICO_PIO_USE_GPIO_BASE is true

Reply:
Good catch, thanks.  I hadn't gotten into the config options quite yet.